test/discharge-wizard/00-required.e2e.spec.js.skip

```diff
@@ -0,0 +1,51 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts.js');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    client
+      .url(`${E2eHelpers.baseUrl}/discharge-upgrade-instructions/`);
+
+    E2eHelpers.overrideSmoothScrolling(client);
+
+    // landing page
+    client
+      .waitForElementVisible('body', Timeouts.normal)
+      .waitForElementVisible('.discharge-wizard', Timeouts.slow)
+      // do not run 'wcag2a' rules because of open aXe bug https://github.com/dequelabs/axe-core/issues/214
+      .axeCheck('.main', { rules: ['section508'] });
```

Comment:
@bshyong any reason why you specified only `section508` rules? Our default rules include `wcag2a` (which we should soon upgrade to `wcag2aa`) but downgrading to remove that set makes it so we can't claim to be WCAG 2.0A compliant so we shouldn't supply a rules attribute

There are some cases where due to problems with the ruleset or code that's not in our control, we whitelist tests that can only comply with `section508`: https://github.com/department-of-veterans-affairs/vets-website/blob/master/test/e2e/sitemap-helpers.js#L25

Reply:
never mind--missed that the aXe bug is referenced in the comments!